src/rabbit_amqqueue.erl

```diff
@@ -609,6 +622,21 @@ consumers_all(VHostPath) ->
                   {ChPid, CTag, AckRequired, Prefetch, Args} <- consumers(Q)]
           end)).
 
+consumers_all(VHostPath, Ref, AggregatorPid) ->
+    ConsumerInfoKeys = consumer_info_keys(),
+    map(list(VHostPath),
+        fun (Q) -> AggregatorPid !
+                       {Ref, get_queue_consumer_info(Q, ConsumerInfoKeys)}
+        end),
+    AggregatorPid ! {Ref, finished},
+    ok.
+
+get_queue_consumer_info(Q, ConsumerInfoKeys) ->
+    lists:flatten([lists:zip(ConsumerInfoKeys,
```

Comment:
@videlalvaro okay i'll update this. had avoided combining any new changes for this PR with the original backend implementations, but will do.